connectivity/mbedtls/platform/inc/platform_mbed.h

```diff
@@ -58,6 +49,17 @@
 #undef MBEDTLS_PSA_CRYPTO_STORAGE_FILE_C
 
 #define MBEDTLS_ENTROPY_HARDWARE_ALT
+
+#endif  // defined(FEATURE_EXPERIMENTAL_API) && defined(FEATURE_PSA)
```

Comment:
The diff rendered by git isn't helpful here. I moved the PSA-related lines above instead of the TRNG lines below, but git decides to show the other way round... Please check the whole file during review.